### PR TITLE
[13.0][MIG] purchase_requisition_tier_validation

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard migration

### CODE REVIEW

File: purchase_requisition_tier_validation/tests/test_purchase_requisition_tier_validation.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestHrExpenseTierValidation(common.TransactionCase):
```
Comment (HviorForgeFlow) [inline purchase_requisition_tier_validation/tests/test_purchase_requisition_tier_validation.py]: Nitpicking

File: purchase_requisition_tier_validation/tests/test_purchase_requisition_tier_validation.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    def setUp(self):
+        super(TestHrExpenseTierValidation, self).setUp()
```
Comment (HviorForgeFlow) [inline purchase_requisition_tier_validation/tests/test_purchase_requisition_tier_validation.py]:
```suggestion
        super().setUp()
```

File: purchase_requisition_tier_validation/views/purchase_requisition_view.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="purchase_requisition_form" model="ir.ui.view">
```
Comment (HviorForgeFlow) [inline purchase_requisition_tier_validation/views/purchase_requisition_view.xml]: Another one...
```suggestion
    <record id="view_purchase_requisition_form" model="ir.ui.view">
```

File: purchase_requisition_tier_validation/views/purchase_requisition_view.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                    <p><i class="fa fa-thumbs-down"/> Operation has been <b>rejected</b>.</p>
+                </div>
+            </header>
+            <xpath expr="//form/div[hasclass('oe_chatter')]" position="before">
+                <field name="review_ids" widget="tier_validation" attrs="{'invisible':[('review_ids', '=', [])]}"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_requisition_filter" model="ir.ui.view">
```
Comment (HviorForgeFlow) [inline purchase_requisition_tier_validation/views/purchase_requisition_view.xml]: Another one...
```suggestion
    <record id="view_purchase_requisition_filter" model="ir.ui.view">
```

File: purchase_requisition_tier_validation/models/tier_definition.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+from odoo import api, models
+
+
+class TierDefinition(models.Model):
+    _inherit = "tier.definition"
+
+    @api.model
+    def _get_tier_validation_model_names(self):
+        res = super(TierDefinition, self)._get_tier_validation_model_names()
```
Comment (HviorForgeFlow) [inline purchase_requisition_tier_validation/models/tier_definition.py]:
```suggestion
        res = super()._get_tier_validation_model_names()
```

**Review by HviorForgeFlow — COMMENTED** [review]
Code suggestions

**Comment by ps-tubtim** [comment]
@HviorForgeFlow Thank you for reviews.

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**